src/main/webapp/js/user-page-loader.js

```diff
@@ -83,21 +84,34 @@ function fetchProfilePic(){
   });
 }
 
+function buildNoPostsDiv(viewingSelf) {
+  var noPostsDiv = document.createElement('div');
+  noPostsDiv.id = 'noPosts';
+  var text = document.createTextNode('The user has no posts yet.');
+  if(viewingSelf){
+    text = document.createTextNode('I have no posts yet.');
```

Comment:
Change I to You
```suggestion
    text = document.createTextNode('You have no posts yet.');
```
In the future, once the new post page is seperated from the user page in #60, I think we should have some call to action here as well. Something like "You have no posts yet, click here to start posting!" or something along those lines.